src/main/java/binnie/core/craftgui/minecraft/EnumColor.java

```diff
@@ -38,7 +34,7 @@ public int getColour() {
 	}
 
 	public String getCode() {
-		return "§" + code;
+		return String.valueOf(code);
```

Comment:
You've changed the behavior. I think you should replace ``` char code ``` with ``` EnumChatFormatting ``` and method should look like: ``` code.toString(); ```
Of course, if you can't get color from ``` EnumChatFormatting ``` or color is never used, then you can replace it with ``` EnumChatFormatting ``` in whole class.